documentation/modules/exploit/unix/webapp/wp_wpdiscuz_unauthenticated_file_upload.md

```diff
@@ -0,0 +1,56 @@
+## Vulnerable Application
+
+This module exploits an arbitrary file upload in the WordPress wpDiscuz plugin
+versions <= `v7.0.4` and >= `v7.0.0`. This flaw gave unauthenticated attackers
```

Comment:
I find the other way around clearer and more consistent with modules/exploits/unix/webapp/wp_wpdiscuz_unauthenticated_file_upload.rb#L20
```diff
-versions <= `v7.0.4` and >= `v7.0.0`.
+versions >= `v7.0.0` and <= `v7.0.4`.
```